### PR TITLE
UBX GPS Reset

### PR DESCRIPTION
**Pull request by LorenzMeier**
This change resets UBX GPS units on boot to ensure that even configured units go back to factory defaults.

### CODE REVIEW

File: src/drivers/gps/gps.cpp [hunk 1]
```diff
@@ -263,6 +263,7 @@ extern "C" __EXPORT int gps_main(int argc, char *argv[]);
 GPS::GPS(const char *path, gps_driver_mode_t mode, GPSHelper::Interface interface, bool fake_gps,
 	 bool enable_sat_info, Instance instance) :
 	_serial_fd(-1),
+	_baudrate(0),
```
Comment (LorenzMeier) [inline src/drivers/gps/gps.cpp]: @bkueng This was your scanning issue. The variable is used as state, but wasn't initialized properly.

**Comment by LorenzMeier** [comment]
This is good to go when it passes CI.

**Comment by dagar** [comment]
There were a couple other uninitialized fields, so I moved to the class definition and did them all.

File: src/drivers/gps/gps.cpp (hunk 1 above)
Comment (bkueng) [reply src/drivers/gps/gps.cpp]: Well, I had additional local changes. The submodule update needed to be in sync with that.

**Pull request merged**